packages/statusbar/src/statusbar.ts

```diff
@@ -112,6 +112,12 @@ export class StatusBar extends Widget implements IStatusBar {
     super.dispose();
   }
 
+  private _isWindowZoomed = () => {
+    // Check is the window width is less than or equal to 630px as this is when the staus bar become unaccessible to the user.
+    // Works across all screen resolution.
+    return window.innerWidth <= 630;
```

Comment:
Question from @gabalafou on the call: is there a justification?

Reply:
This number is because this is when the status bar starts to become inaccessible. The comment above has been reviewed by @gabalafou and changed to make the reasoning of this number clearer.